fe/fe-core/src/main/java/org/apache/doris/statistics/Statistics.java

```diff
@@ -119,7 +119,8 @@ public void normalizeColumnStatistics(double inputRowCount, boolean isNumNullsDe
             // the following columnStatistic.isUnKnown() judgment is loop inside since current doris
             // supports partial stats deriving, i.e, allowing part of tables have stats and other parts don't,
             // or part of columns have stats but other parts don't, especially join and filter estimation.
-            if (!checkColumnStatsValid(columnStatistic, rowCount) && !columnStatistic.isUnKnown()) {
+            if ((!checkColumnStatsValid(columnStatistic, rowCount) && !columnStatistic.isUnKnown())
+                    || isNumNullsDecreaseByProportion && columnStatistic.numNulls != 0) {
```

Comment:
if col stats is unknown, do not need normalize
the condition should be:
!columnStatistic.isUnKnown() && (!checkColumnStatsValid(columnStatistic, rowCount) || !checkColumnStatsValid(columnStatistic, rowCount))

Reply:
that's right, thx for the suggestion